### PR TITLE
[misc] Refactoring & Cleanup - aliases and priorities in src/lib/units/

### PR DESCRIPTION
**Pull request by qzoke**
My mission with this refactoring was to make it more readable and procedural. Not only it reduces 32 function calls, It's all in one place. All aliases in aliases.js file and all priorities in priorities.js file

1. All the functions which were responsible for creating aliases from following files(comments are file names) have been moved to src/lib/units/aliases.js

// src/lib/units/day-of-month.js
addUnitAlias('date', 'D');
// src/lib/units/day-of-week.js
addUnitAlias('day', 'd');
addUnitAlias('weekday', 'e');
addUnitAlias('isoWeekday', 'E');
// src/lib/units/day-of-year.js
addUnitAlias('dayOfYear', 'DDD');
// src/lib/units/hour.js
addUnitAlias('hour', 'h');
// src/lib/units/millisecond.js
addUnitAlias('millisecond', 'ms');
// src/lib/units/minute.js
addUnitAlias('minute', 'm');
// src/lib/units/month.js
addUnitAlias('month', 'M');
// src/lib/units/quarter.js
addUnitAlias('quarter', 'Q');
// src/lib/units/second.js
addUnitAlias('second', 's');
// src/lib/units/week-year.js
addUnitAlias('weekYear', 'gg');
addUnitAlias('isoWeekYear', 'GG');
// src/lib/units/week.js
addUnitAlias('week', 'w');
addUnitAlias('isoWeek', 'W');
// src/lib/units/year.js
addUnitAlias('year', 'y');

2. Ran all the functions and final result has been saved as "aliases" object in  src/lib/units/aliases.js.

3. Commented out all the aliases creation functions in src/lib/units/aliases.js as we have final and known Object as a variable.

4. All the functions which were responsible for creating priorities from following files(comments are file names) have been moved to src/lib/units/priorities.js

// src/lib/units/day-of-month.js
addUnitPriority('date', 9);
// src/lib/units/day-of-week.js
addUnitPriority('day', 11);
addUnitPriority('weekday', 11);
addUnitPriority('isoWeekday', 11);
// src/lib/units/day-of-year.js
addUnitPriority('dayOfYear', 4);
// src/lib/units/hour.js
addUnitPriority('hour', 13);
// src/lib/units/millisecond.js
addUnitPriority('millisecond', 16);
// src/lib/units/minute.js
addUnitPriority('minute', 14);
// src/lib/units/month.js
addUnitPriority('month', 8);
// src/lib/units/quarter.js
addUnitPriority('quarter', 7);
// src/lib/units/second.js
addUnitPriority('second', 15);
// src/lib/units/week-year.js
addUnitPriority('weekYear', 1);
addUnitPriority('isoWeekYear', 1);
// src/lib/units/week.js
addUnitPriority('week', 5);
addUnitPriority('isoWeek', 5);
// src/lib/units/year.js
addUnitPriority('year', 1);

5. Ran all the functions and final result has been saved as "priorities" object in  src/lib/units/priorities.js.

6. Commented out all the priorities creation functions in src/lib/units/aliases.js as we have final and known Object as a variable.

### CODE REVIEW

File: src/lib/units/priorities.js [hunk 1]
```diff
@@ -1,11 +1,64 @@
...
+addUnitPriority('quarter', 7);
+//second.js
+addUnitPriority('second', 15);
+//week-year.js
+addUnitPriority('weekYear', 1);
+addUnitPriority('isoWeekYear', 1);
+//week.js
+addUnitPriority('week', 5);
+addUnitPriority('isoWeek', 5);
+//year.js
```
Comment (anandfresh) [inline src/lib/units/priorities.js:56]: Can we remove these commented codes ?

File: src/lib/units/aliases.js [hunk 2]
```diff
@@ -1,12 +1,96 @@
...
+//minute.js
+addUnitAlias('minute', 'm');
+//month.js
+addUnitAlias('month', 'M');
+//quarter.js
+addUnitAlias('quarter', 'Q');
+//second.js
+addUnitAlias('second', 's');
+//week-year.js
+addUnitAlias('weekYear', 'gg');
```
Comment (anandfresh) [inline src/lib/units/aliases.js:85]: can we remove these commented line of codes ?

**Review by anandfresh — CHANGES_REQUESTED** [review]

**Comment by qzoke** [comment]
@anandfresh Helllo sir,
Sorry for late reply. I didn't knew that you are a moderator. 

 I removed comments as requested..

**Review by anandfresh — APPROVED** [review]

**Comment by ichernev** [comment]
Merge in d6d5df7b

**Pull request closed without merging**